src/IMenuGraphr.ts

```diff
@@ -263,6 +263,11 @@ export interface IMenuSchema extends IMenuBase {
      * How the menu should be positioned within its container.
      */
     position?: IMenuSchemaPosition;
+
+    /**
+     * Whether or not the last selected index should be saved.
+     */
+    saveIndex?: boolean;
```

Comment:
Yeah, there was an error in `FSP/src/settings/menus.ts` where it would not recognize the property when it was `IListMenuSchema` so I put it in `IMenuSchema`